lib/src/components/NxDropdownMenu/NxDropdownMenu.scss

```diff
@@ -85,7 +85,11 @@
 
 .nx-dropdown-link {
   color: var(--nx-color-link);
+  margin: var(--nx-spacing-base) var(--nx-spacing-4x);
+  max-width: calc(100% - var(--nx-spacing-8x));
+  padding: 0;
   text-decoration: underline;
+  width: fit-content;
```

Comment:
What's this for? I don't see a meaningful difference with it

Reply:
> What's this for? I don't see a meaningful difference with it

Oh the width of the focus box.  Let's use `min-content` instead, as it works just as well and `fit-content` is much more complex to think about.

Reply:
Updated here a8e7a0c7b241020d6dce431d470a18c1bc5666cc.